vendor-extra/LiberoPatternsBundle/src/Resources/views/italic.html.twig

```diff
@@ -0,0 +1,9 @@
+{% spaceless %}
+
+  <i {%- include '@LiberoPatterns/attributes.html.twig' %}>
```

Comment:
is the `{%-` supposed to strip the `<i ` space or can in principle remain as `<i >` in the empty attributes case? Still valid I think

Reply:
This comes from the pattern library, but it will strip it. The included template prepends a space if it has a value. Meant to make the template easier to read without resulting in `<i >`.